pkg/kubelet/cm/devicemanager/plugin/v1beta1/stub.go

```diff
@@ -84,12 +96,15 @@ func NewDevicePluginStub(devs []*pluginapi.Device, socket string, name string, p
 		resourceName:               name,
 		preStartContainerFlag:      preStartContainerFlag,
 		getPreferredAllocationFlag: getPreferredAllocationFlag,
+		registerControlFunc:        defaultRegisterControlFunc,
 
 		stop:   make(chan interface{}),
 		update: make(chan []*pluginapi.Device),
 
 		allocFunc:             defaultAllocFunc,
 		getPreferredAllocFunc: defaultGetPreferredAllocFunc,
+		termSignal:            make(chan bool, 1),
```

Comment:
I see you writing to this channel, but I don't see it being consumed anywhere.

Reply:
Thanks for catching this. I had added this with the understanding that `termSignal` would be needed for handling termination within ListAndWatch() but later realized we already have `stop` channel for that and missed removing this. I will update.